listenbrainz/webserver/__init__.py

```diff
@@ -149,7 +149,7 @@ def create_app(config_path=None, debug=None):
     app = gen_app(config_path=config_path, debug=debug)
 
     # Static files
-    import listenbrainz.webserver.static_manager
+    import listenbrainz.webserver.static_manager as static_manager
```

Comment:
why was this change needed?

Reply:
There seems to be some bug with `static_manager`,  especially the `read_manifest` function. I thought the above change would fix the error. The change  worked once but did not work the next time. I committed the change by mistake. I will look into the bug and make appropriate changes, till then I am rolling back to the previous version without `as static_manger`.